ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -198,11 +215,27 @@ const KVMResourcesCard = ({
         >
           <h4 className="p-heading--small">Total VMs</h4>
           <ContextualMenu
-            dropdownContent={<></>}
+            dropdownClassName="kvm-machine-list-modal"
+            dropdownContent={
+              <MachineListTable
+                hiddenColumns={[
```

Comment:
I think we should probably add a `className` prop to `MachineListTable` so we can give it custom cell widths. Right now the status column is far too small.